File: MotivApp/app/src/main/java/com/motiv/piotr/UserDetailActivity.java
package com.motiv.piotr;

import android.widget.ImageView;
import android.widget.LinearLayout;
import android.widget.TextView;
import androidx.annotation.Nullable;
import androidx.appcompat.app.AppCompatActivity;
import androidx.databinding.DataBindingUtil;
import com.motiv.piotr.dao.DaoRepository;
import com.motiv.piotr.dao.DaoRepositoryFactory;
import com.motiv.piotr.dao.LocalStorage;
import com.motiv.piotr.databinding.UserdetailactivityBinding;
import com.squareup.picasso.Picasso;

public class UserDetailActivity extends AppCompatActivity {

    private UserdetailactivityBinding userdetailactivityBinding;
    private com.motiv.piotr.User user;
    private UsersListAdapter usersListAdapter;
    private PostsListAdapter postsListAdapter;
    private PhotosPagerAdapter photosPagerAdapter;
    private FragmentsPagerAdapter fragmentsPagerAdapter;
    private GoRestApi goRestApi;
    private DaoRepository daoRepository;
    private LocalStorage localStorage;
    private NavigationController navigationController;
    private LinearLayout linearlayout00;
    private ImageView imageview10;
    private TextView textview11;

    @Override
    protected void onCreate(@Nullable android.os.Bundle savedInstanceState) {

        super.onCreate(savedInstanceState);
        userdetailactivityBinding =
                DataBindingUtil.setContentView(this, R.layout.userdetailactivity);

        user = com.motiv.piotr.User.fromJson(getIntent().getStringExtra("user"));

        usersListAdapter = new UsersListAdapter();
        postsListAdapter = new PostsListAdapter();
        photosPagerAdapter = new PhotosPagerAdapter();
        fragmentsPagerAdapter =
                new FragmentsPagerAdapter(UserDetailActivity.this.getSupportFragmentManager());
        daoRepository = DaoRepositoryFactory.getInstance(UserDetailActivity.this);
        localStorage = LocalStorage.getInstance(UserDetailActivity.this);
        navigationController = new NavigationController(UserDetailActivity.this);
        goRestApi = GoRestApiFactory.getInstance(localStorage);
        linearlayout00 = userdetailactivityBinding.linearlayout00;
        imageview10 = userdetailactivityBinding.imageview10;
        textview11 = userdetailactivityBinding.textview11;

        Picasso.with(UserDetailActivity.this)
                .load(user.getLinks().getAvatar().getHref())
                .into(imageview10);
        ;
        textview11.setText(user.getFirst_name());
    }
}

File: MotivApp/app/src/main/java/com/motiv/piotr/UserDetailActivity.kt
 
package com.motiv.piotr
import android.os.Bundle
import android.widget.ImageView
import android.widget.LinearLayout
import android.widget.TextView
import androidx.appcompat.app.AppCompatActivity
import com.motiv.piotr.dao.DaoRepository
import com.motiv.piotr.dao.DaoRepositoryFactory
import com.motiv.piotr.dao.LocalStorage
import com.squareup.picasso.Picasso
import kotlinx.android.synthetic.main.userdetailactivity.*

public class UserDetailActivity : AppCompatActivity() {

    private lateinit var user: com.motiv.piotr.User

    private lateinit var usersListAdapter: UsersListAdapter

    private lateinit var postsListAdapter: PostsListAdapter

    private lateinit var photosPagerAdapter: PhotosPagerAdapter

    private lateinit var fragmentsPagerAdapter: FragmentsPagerAdapter

    private lateinit var goRestApi: GoRestApi

    private lateinit var daoRepository: DaoRepository

    private lateinit var localStorage: LocalStorage

    private lateinit var navigationController: NavigationController

    private lateinit var linearlayout00: LinearLayout

    private lateinit var imageview10: ImageView

    private lateinit var textview11: TextView

    override fun onCreate(savedInstanceState: android.os.Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.userdetailactivity)

        user = com.motiv.piotr.User.fromJson(getIntent().getStringExtra("user"))

        usersListAdapter = UsersListAdapter()
        postsListAdapter = PostsListAdapter()
        photosPagerAdapter = PhotosPagerAdapter()
        fragmentsPagerAdapter = FragmentsPagerAdapter(this@UserDetailActivity.getSupportFragmentManager())
        daoRepository = DaoRepositoryFactory.getInstance(this@UserDetailActivity)
        localStorage = LocalStorage.getInstance(this@UserDetailActivity)
        navigationController = NavigationController(this@UserDetailActivity)
        goRestApi = GoRestApiFactory.getInstance(localStorage)
        linearlayout00 = findViewById<LinearLayout>(R.id.linearlayout00)
        imageview10 = findViewById<ImageView>(R.id.imageview10)
        textview11 = findViewById<TextView>(R.id.textview11)

        Picasso.with(this@UserDetailActivity).load(user.getLinks().getAvatar().getHref()).into(imageview10)
        textview11.setText(user.getFirst_name())
    }
}
